### PR TITLE
Fix some alex warnings

### PR DESCRIPTION
**Pull request by Licenser**

### CODE REVIEW

**Review by mfelsche — DISMISSED** [review]
LGTM

**Review by mfelsche — DISMISSED** [review]
still LGTM

**Review by darach — DISMISSED** [review]
LGTM

File: blog/2021-07-06-Blog-Rohit-Property-Based-Testing.md [hunk 1]
```diff
@@ -23,25 +23,26 @@ My work involved writing "Property-based tests for tremor-script" and some of th
...
 
 - Tremor-script
-    - It is an interpreted expression-oriented language designed for the filtering, extraction, transformation and streaming of structured data in a stream or event-based processing system which is explicitly turing incomplete used to easily write programs specific to Tremor use-cases.
+    - It is an interpreted expression-oriented language designed for the filtering, extraction, transformation and streaming of structured data in a stream or event-based processing system which is explicitly turing incomplete used to write programs specific to Tremor use-cases.
     - It is written using Rust but tested using Erlang
 - Property based testing
     - We have seen various types of testing approaches like Unit testing, Integration testing, End2End testing etc.
     - The purpose of tests is to check if our code is failing anywhere and test the same with various inputs.
-    - Fig 1 shows different types of tests to easily understand which features are specialised in what.
+    - Fig 1 shows different types of tests to understand which features are specialised in what.
```
Comment (skoech) [inline blog/2021-07-06-Blog-Rohit-Property-Based-Testing.md:39]: How about  ... tests to *help* understand...?

File: docs/Artefacts/offramps.md [hunk 2]
```diff
@@ -1002,7 +1004,7 @@ Example:
       dir: "temp/kv" # directory to store data in
 ```
 
-One response event is sent for every command in the handled event. On success to the `out` port, in the error case (invalid format or error during kv operation) to the `err` port.
+One response event is sent for every command in the handled event. On success to the `out` port, in the error case (malformed or error during kv operation) to the `err` port.
```
Comment (skoech) [inline docs/Artefacts/offramps.md:1007]: I think you meant malformed *format* here.

Comment (Licenser) [reply docs/Artefacts/offramps.md:1007]: I changed it to `malformed data` which seems to fit best :)

File: blog/2021-07-06-Blog-Rohit-Property-Based-Testing.md (hunk 1 above)
Comment (Licenser) [reply blog/2021-07-06-Blog-Rohit-Property-Based-Testing.md:39]: good idea :)

**Review by skoech — COMMENTED** [review]
Looks good. *Just* a few suggestions, maybe. :D

**Review by skoech — APPROVED** [review]
Awesome!

**Pull request merged**